test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -6,7 +6,7 @@ def setup
 
     @gateway = CyberSourceGateway.new({nexus: "NC"}.merge(fixtures(:cyber_source)))
 
-    @credit_card = credit_card('4111111111111111')
+    @credit_card = credit_card('4111111111111111', verification_value: '321')
```

Comment:
``assert_scrubbed`` finds ``123`` from the email if we don't use a different cvv here